python/mxnet/__init__.py

```diff
@@ -87,6 +86,9 @@
 
 from . import gluon
 
+# Dynamic library module should be done after ndarray and symbol are initialized
+from . import library
```

Comment:
Can we also add this namespace to the doc: https://mxnet.apache.org/api/python/docs/api/ ?

Reply:
@eric-haibin-lin We are not working on docs/tutorials in this PR. Ive added an entry to the PR description in the "Future/Next-steps" section that we'll address in the next PR.